front/src/applications/editor/Editor.tsx

```diff
@@ -139,6 +141,7 @@ const Editor = () => {
       dispatch,
       editorState,
       infraID,
+      isLocked,
```

Comment:
You should rename it (right now it is not used in the left panels, so if you select a track for instance, you can still save it)
```suggestion
      infraIsLocked: isLocked,
```

Reply:
fixed